vmdb/app/models/configured_system_foreman.rb

```diff
@@ -1,2 +1,13 @@
 class ConfiguredSystemForeman < ConfiguredSystem
+  include ProviderObjectMixin
+
+  def provider_object(connection = nil)
+    (connection || raw_connect).host_ref(manager_ref)
```

Comment:
Maybe just `host` for the method name?  Less happy with `host_obj`, but I can't think of an alternative.